packages/nyc-test-coverage/src/index.ts

```diff
@@ -28,6 +69,26 @@ export class WorkflowCoverage {
     };
   }
 
+  /**
+   * Modify the given Worker config to auto instrument Workflow
+   * code using istanbul-instrumenter-loader
+   */
+
+  webpackConfigHook(workflowsPath: string, config: WebpackConfigHookType): WebpackConfigHookType {
```

Comment:
Calling this `webpackConfigHook` confuses me with `WorkerOptions.webpackConfigHook`. What about `addInstrumenterRule`?

Reply:
That's a fair point. I called this `webpackConfigHook` originally because you were supposed to add this method to the Webpack config yourself. But then I needed to add `workflowsPath`. It makes sense to rename now that this method is no longer the same function signature as the `webpackConfigHook` option